hydromt_wflow/workflows/soilgrids.py

```diff
@@ -764,16 +764,16 @@ def soilgrids_sediment(
 
     The following soil parameter maps are calculated:
 
-    * fclay_soil: clay content of the topsoil [g/g]
-    * fsilt_soil: silt content of the topsoil [g/g]
-    * fsand_soil: sand content of the topsoil [g/g]
-    * fsagg_soil: small aggregate content of the topsoil [g/g]
-    * flagg_soil: large aggregate content of the topsoil [g/g]
+    * soil_clay_fraction: clay content of the topsoil [g/g]
+    * soil_silt_fraction: silt content of the topsoil [g/g]
+    * soil_sand_fraction: sand content of the topsoil [g/g]
+    * soil_sagg_fraction: small aggregate content of the topsoil [g/g]
+    * soil_lagg_fraction: large aggregate content of the topsoil [g/g]
     * soil_detachability: mean detachability of the soil (Morgan et al., 1998) [g/J]
-    * usle_k: soil erodibility factor from the USLE equation [-]
-    * d50_soil: median sediment diameter of the soil [mm]
-    * c_govers: Govers factor for overland flow transport capacity [-]
-    * n_govers: Govers exponent for overland flow transport capacity [-]
+    * erosion_usle_k: soil erodibility factor from the USLE equation [-]
```

Comment:
If there is a hydromt name, better to use it here in workflows. Renaming will happen in wflow_sediment.py before calling set_grid
```suggestion
    * usle_k: soil erodibility factor from the USLE equation [-]
```